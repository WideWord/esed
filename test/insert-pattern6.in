Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
Vivamus congue mauris arcu, vitae volutpat turpis semper sed. 
Lorem ipsum dolor sit amet. 
Mauris et tincidunt nibh, sit amet semper lacus. 
Pellentesque habitant morbi tristique senectus et netus.
Lorem ipsum rare dolor sit amet. 