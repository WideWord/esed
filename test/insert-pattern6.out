Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
Hello
Vivamus congue mauris arcu, vitae volutpat turpis semper sed. 
Lorem ipsum dolor sit amet. 
Hello
Mauris et tincidunt nibh, sit amet semper lacus. 
Pellentesque habitant morbi tristique senectus et netus.
Lorem ipsum rare dolor sit amet. 